### PR TITLE
refactor(Entity): Provide Update at top level import, closes #670

### LINKED ISSUE #670: Entity: Update type should be provided from top level import.
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[x] Feature request
[ ] Documentation issue or request
</code></pre>

## What is the current behavior?

Currently Update type for type checking updateMany and updateOne methods of entity adapter is only available in deep import `import {Update} from '@ngrx/entity/src/models'`
## Expected behavior:

It should be provided as top level import

```ts
      import {Update} from '@ngrx/entity';

      // ...
      const updates: Update[] = action.payload.map(item => ({id: item.unixTime, changes: item}));
      return {
        ...adapter.updateMany(updates, state),
        loading: false,
        loaded: true,
        errorMess: null,
      };

```

### PR DESCRIPTION
**Pull request by sandangel**
Reference: #670

### CODE REVIEW

File: docs/entity/adapter.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ export interface User {
 ```ts
 import { Action } from '@ngrx/store';
 import { User } from './user.model';
+import { Update } from '@ngrx/entity';
```
Comment (brandonroberts) [inline docs/entity/adapter.md]: Minor nit but will you move this import above the `user.model` import?

**Comment by brandonroberts** [comment]
@sandangel you want to take a stab at updating the entity schematic also?

**Comment by sandangel** [comment]
I will take a look

**Comment by sandangel** [comment]
Can you review @brandonroberts

**Comment by brandonroberts** [comment]
Nice work, thanks!

**Pull request merged**